ep_transition/transition_run_thread.py

```diff
@@ -23,9 +23,17 @@ class TransitionRunThread(threading.Thread):
     :ivar std_err: The standard error output from the transition process
     """
 
-    def __init__(self, transitions_to_run: List[TransitionBinary], working_directory: Path, original_file_path: Path,
-                 keep_old: bool, ready_callback: Callable, increment_callback: Callable,
-                 msg_callback: Callable, done_callback: Callable):
+    def __init__(
```

Comment:
I assume the policy is "if all function arguments fit on one line, use one line, otherwise one argument per line".  That's fine, just another thing to get used to.

Reply:
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#how-black-wraps-lines